app/routes/farmer-apply/species-eligibility.js

```diff
@@ -0,0 +1,63 @@
+const Joi = require('joi')
+const { farmerApplyData: { eligibleSpecies } } = require('../../session/keys')
+const { getYesNoRadios } = require('../helpers/yes-no-radios')
+const session = require('../../session')
+const speciesTypes = require('../../constants/species')
+const speciesContent = require('../../constants/species-content')
+const backLink = '/farmer-apply/which-review'
+const radioId = 'eligibleSpecies'
```

Comment:
You can use the `eligibleSpecies` session key in place of creating this new var.